app/components/article-featured-video.js

```diff
@@ -23,16 +23,27 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
 
 	classNames: ['article-featured-video'],
 
+	// transparent gif
+	attributionAvatarUrl: 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==',
+
 	smartBannerVisible: readOnly('smartBanner.smartBannerVisible'),
 	isFandomAppSmartBannerVisible: readOnly('smartBanner.isFandomAppSmartBannerVisible'),
 
+	initialVideoDetails: readOnly('model.embed.jsParams.playlist.0'),
+	currentVideoDetails: oneWay('initialVideoDetails'),
 	metadata: reads('model.metadata'),
-	placeholderImage: readOnly('model.embed.jsParams.playlist.0.image'),
+	placeholderImage: readOnly('initialVideoDetails.image'),
 
 	placeholderStyle: computed('placeholderImage', function () {
 		return htmlSafe(`background-image: url(${this.get('placeholderImage')})`);
 	}),
 
+	hasAttribution: computed('currentVideoDetails.{username,userUrl,userAvatarUrl}', function () {
```

Comment:
https://emberjs.com/api/ember/2.18/classes/@ember%2Fobject%2Fcomputed/methods/and?anchor=and